code/modules/mob/living/basic/space_fauna/bear/_bear.dm

```diff
@@ -94,12 +94,11 @@
 /mob/living/basic/bear/snow/ancient
 	name = "ancient polar bear"
 	desc = "A grizzled old polar bear, its hide thick enough to make it impervious to almost all weapons."
-	status_flags = CANPUSH | GODMODE
 	gold_core_spawnable = NO_SPAWN
 
 /mob/living/basic/bear/snow/Initialize(mapload)
```

Comment:
Moved godmode from bear/snow/ancient to bear/snow

Reply:
OOOOH FUK